Source/Core/RectangleGeometry.js

```diff
@@ -775,5 +775,27 @@ define([
         });
     };
 
+    RectangleGeometry._createShadowVolume = function(rectangleGeometry, minHeightFunc, maxHeightFunc) {
+        var granularity = rectangleGeometry._granularity;
+        var ellipsoid = rectangleGeometry._ellipsoid;
+
+        var minHeight = minHeightFunc(granularity, ellipsoid);
+        var maxHeight = maxHeightFunc(granularity, ellipsoid);
+
+        // TODO: stRotation
```

Comment:
I was going to leave the TODOs there until texturing is implemented. Do we want to support rotated textures?